### PR TITLE
Nit use credentials

### PR DESCRIPTION
**Pull request by subodh101**
What?
As per the title.

Why?
https://github.com/kubeflow/pipelines/blob/ccc763fb718569a21b95a107ffe5beb023334742/sdk/python/kfp/_auth.py#L32

```
LOCAL_KFP_CREDENTIAL = os.path.expanduser('~/.config/kfp/credentials.json')
```

### CODE REVIEW

**Comment by 8bitmp3** [comment]
/assign @Bobgy @IronPan @neuromage @Ark-kun @numerology

PTAL 🔍 

Thank you all

File: content/en/docs/pipelines/multi-user.md [hunk 1]
```diff
@@ -77,11 +77,11 @@ print(client.list_runs(namespace='<Your namespace>'))
 To store your user namespace as the default context, use the
 [`set_user_namespace`](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.client.html#kfp.Client.set_user_namespace)
 method. This method stores your user namespace in a configuration file at
-`$HOME/.config/kfp/context.json`. After setting a default namespace, the SDK
+`$HOME/.config/kfp/credentials.json`. After setting a default namespace, the SDK
 methods default to use this namespace if no namespace argument is provided.
 
 ```python
-# Note, this saves the namespace in `$HOME/.config/kfp/context.json`. Therefore,
+# Note, this saves the namespace in `$HOME/.config/kfp/credentials.json`. Therefore,
```
Comment (Bobgy) [inline content/en/docs/pipelines/multi-user.md:84]: I think this is right: https://github.com/kubeflow/pipelines/blob/e3992faf8305570ca56c5932d701eb8765305568/sdk/python/kfp/_client.py#L118

File: content/en/docs/pipelines/multi-user.md [hunk 2]
```diff
@@ -77,11 +77,11 @@ print(client.list_runs(namespace='<Your namespace>'))
 To store your user namespace as the default context, use the
 [`set_user_namespace`](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.client.html#kfp.Client.set_user_namespace)
 method. This method stores your user namespace in a configuration file at
-`$HOME/.config/kfp/context.json`. After setting a default namespace, the SDK
+`$HOME/.config/kfp/credentials.json`. After setting a default namespace, the SDK
```
Comment (Bobgy) [inline content/en/docs/pipelines/multi-user.md:80]: Thanks for finding the problem!

**Review by Bobgy — COMMENTED** [review]
Hi @subodh101, thank you for the fix! Please see comments

File: content/en/docs/pipelines/multi-user.md (hunk 2 above)
Comment (Bobgy) [reply content/en/docs/pipelines/multi-user.md:80]: Well, no, I think this is right too

**Comment by 8bitmp3** [comment]
@subodh101 Hi, any update this?

cc @Bobgy (left comments above), @RFMVasconcelos

**Comment by subodh101** [comment]
Thank you.
Closing it for now. I thought it was a nit.

/close

**Pull request closed without merging**